src/utils/formatLocaleNumbers.js

```diff
@@ -0,0 +1,9 @@
+// Overwrites the default Persian numbering of the Farsi language to use Hindu-Arabic numerals (0-9)
+// Context: https://github.com/ethereum/ethereum-org-website/pull/5490#pullrequestreview-892596553
+export const formatLocaleNumbers = (locale) => {
```

Comment:
Not opposed to a new file but we do have an existing `translations.js` utils file that I feel like this function could fit within, given it deals with language formatting.

Reply:
Also given I'm already nitpicking 😅 I think the name could be a bit clearer - it's slightly misleading since this function doesn't actually format anything. Maybe something like `getLocaleNumberFormat`?

Reply:
> Not opposed to a new file but we do have an existing `translations.js` utils file that I feel like this function could fit within, given it deals with language formatting.

Good point. Moved it in there.

> Also given I'm already nitpicking 😅 I think the name could be a bit clearer - it's slightly misleading since this function doesn't actually format anything. Maybe something like getLocaleNumberFormat?

Not nitpicking at all it was a bad name 😬 updated it to `getLocaleForNumberFormat`.